packages/transform-to-vanilla/src/transform-keys/merge-key.ts

```diff
@@ -0,0 +1,26 @@
+const RegExp = /[$_]$/;
+
+// == Interface ================================================================
+export function removeSignSimbol(keyStr: string) {
```

Comment:
You should be able to distinguish between input by utilizing a template literal string.
This is to infer that when the last letter of a key is `$` or `_`, it has an array value.

For the return value, I think `string` is fine.
(If there's a problem later, we'll look into it further.)

Reply:
Oh i see, then im gonna fix to better way.
Thank you for advice!